src/modal/modal.component.ts

```diff
@@ -129,6 +129,11 @@ export class Modal implements AfterViewInit, OnInit, OnDestroy {
 	 */
 	@Input() modalLabel = "default";
 
+	/**
+	 * Specify whether the modal contains scrolling content
+	 */
+	@Input() hasScrollingContent = false;
```

Comment:
We'll need to expose this to the alert modal ... also we'll need to consider how this affects users that are now taking advantage of the new default (hasScrollingContent = true) ...

Reply:
Do you think that it should be defaulted to true then?